sdk/communication/Azure.Communication.Chat/tests/ChatClients/ChatClientsTest.cs

```diff
@@ -147,6 +148,8 @@ public void E2E_ThreadCreateUpdateGetDelete_MemberAddUpdateRemove_MessageGetSend
             chatClient.DeleteChatThread(threadId);
             #endregion Snippet:Azure_Communication_Chat_Tests_E2E_DeleteChatThread
 
+            Thread.Sleep(500);
```

Comment:
@minnieliu These are synchronous calls, we should not be adding Thread.Sleep on our tests. Any reason we need this ?

Reply:
This test is flakey in the pipeline, see this: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=559498&view=logs&j=fb683405-d979-52da-6de9-2541dff429a6&t=47b32f90-f4d1-5232-a550-dd1152acf48a 

In Java, a 500ms sleep was added to the tests. Let me know if there is a better solution @juancamilor

Reply:
The thread sleep here is not doing much actually. This value was taken before :
var readReceiptsCount = readReceipts.Count();
Probably what we are seeing here is a case where the read receipt is sent, and downstream on MFE it gets stored, but due to distributed storage when we do a get it is probably not there yet (we will confirm that soon). I suggest we just remove the thread sleeps and comment the readreciept assertion.